### ISSUE TITLE
Powah: Lens of the ender creation doesn't work

### ISSUE THREAD

**Issue by groverfield** [NillerMedDild/Enigmatica6#2254]
**Modpack version**
Tested in Version 0.4.14

**Describe the bug**
A "Lens of the ender" is only created if you have multiple photoelectric panes, or a Lens of the ender already in your inventory.  The quest will still complete, but the item either doesn't generate, or disappears.  Testing for this was done in the end, as dimensions might matter.

**Comment by NillerMedDild** [comment]
Thanks for the report. This is technically a duplicate issue, but you've provided some new info so I'll have another go at testing it 🙂

**Comment by NillerMedDild** [comment]
I reproduced it the steps you provided, thank you :D

**Comment by Splutty** [comment]
Will this ever be resolved, because it's still an issue, and it's extremely annoying :P

**Comment by Darkere** [comment]
As far as I know Powah dev is MIA... so probably not.

**Comment by Splutty** [comment]
Well crap :P Just ran into this earlier, so definitely not fixed.

**Comment by MuteTiefling** [comment]
Seems to happen completely randomly. I made one fine the other day, then had one fail just the next day.

Probably best to just make a simple recipe for them. Pearl and photovoltaic in a pressure chamber for expert, and just a 3x3 craft for normal...

**Comment by Splutty** [comment]
Grover's findings at 100% reproducable by me.

1 Photo: It disappears.
2 Photo, create a Lens, leaves a Photo.
1 Photo, 1 Lens, creates another Lens.
1 Photo, no Lens, see point 1.

**Comment by MuteTiefling** [comment]
Fair point. I had forgotten the glyph and was mostly thinking of when you can make the rest of the components.

**Comment by leagris** [comment]
Still not fixed in Powah 5.0.5 need have two panes in the same slot when right-clicking Enderman/Endermite. Please re-open
